src/MCPServer/share/mysql_dev_9647_archivesspace_inherit_notes.sql

```diff
@@ -0,0 +1,5 @@
+-- Add column inherit_notes column to administration_archivesspaceconfig
+ALTER TABLE `administration_archivesspaceconfig` ADD COLUMN `inherit_notes` bool NOT NULL AFTER `repository`;
```

Comment:
I've noticed that syncdb populates the column as inherit_notes tinyint(1) NOT NULL, so I've updated my migration script to update the existing row so inherit_notes equals True. I guess that under normal circumstances the according Django migration would do that for me?